libs/search-widget/src/common/paragraph-result/ParagraphResult.svelte

```diff
@@ -82,9 +85,11 @@
     </div>
     <div
       class="paragraph-text"
-      class:ellipsis={ellipsis && !expanded}
+      class:ellipsis={hasEllipsis && !expanded}
       bind:this={paragraphElement}>
-      {@html paragraph.text}
+      <MarkdownRendering
```

Comment:
is it ok to render all paragraphs as markdown ? or should only the fields with "KEEP_MARKDOWN" format be rendered ?